package.json

```diff
@@ -112,6 +116,9 @@
     "ci": "yarn install && yarn run clean && yarn run build-rb && yarn run lint && yarn run test && yarn run build-ci",
     "clean": "rm -rf dist && rm -rf reports/coverage && rm -rf src/i18n/json",
     "commitmsg": "commitlint -e",
+    "cy:open": "yarn cy:wait; yarn cypress open",
+    "cy:run": "yarn cy:wait; yarn cypress run --spec test/integration/**/*.test.js",
+    "cy:wait": "wait-on tcp:8000",
```

Comment:
The associated box-ui-elements script was recently improved. Probably want to take a look and use the same concept here.